pyccel/codegen/printing/ccode.py

```diff
@@ -368,6 +380,47 @@ def _print_PyccelPow(self, expr):
     def _print_Import(self, expr):
         return '#include <{0}>'.format(expr.source)
 
+    def _print_String(self, expr):
+        return "\"{}\"".format(expr.arg)
+
+    def _print_Print(self, expr):
+        self._additional_imports.add("stdio.h")
+        type_to_format = {('real',8)    : '%lf',
+                          ('real',4)    : '%f',
+                          ('complex',8) : '(%lf + %lfj)',
+                          ('complex',4) : '(%f + %fj)',
+                          ('int',4)     : '%d',
+                          ('int',8)     : '%ld',
+                          ('int',2)     : '%hd',
+                          ('int',1)     : '%c',
+                          ('bool',4)    : '%s',
+                          ('string', 0) : '%s'}
+        args_format = []
+        args = []
+        end = '\\n'
+        sep = ' '
+        for f in expr.expr:
+            if isinstance(f, ValuedVariable):
+                if f.name == 'sep'      :   sep = str(f.value)
+                elif f.name == 'end'    :   end = str(f.value)
+            else:
+                try:
+                    args_format.append(type_to_format[(self._print(f.dtype), f.precision)])
+                except KeyError:
+                    raise ("{} type is not supported currently".format(f.dtype))
```

Comment:
Please use `errors.report` to raise errors. This will ensure that we raise a `PyccelError` rather than a `KeyError` and it will potentially allow us to provide more information about the location of the problem in the future

Reply:
done.